### DISCUSSION TITLE
Default questionnaire not change

### DISCUSSION THREAD

**Discussion by Gianca77** [globaleaks/globaleaks-whistleblowing-software#3540] [Q&A] [answered]
### What version of GlobaLeaks are you using?

4.11.5

### What browser(s) are you seeing the problem on?

Chrome

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

I duplicated the default questionnaires (GLOBALEAKS), made some changes, and renamed xxx.
I set the new questionnaires in Settings -> Advanced -> Default questionnaire but every time I load my webpage it appears with the original template.
What can I do? Maybe I miss setting something? I read the documentation but I didn't find nothing

**Comment by evilaliv3** [comment, marked as answer]
@Gianca77: The default questionnaire option that you are using make it possible that if you create a new channel the new channel is created by using the default questionnaire.

I think that instead you missed to assign your custom questionnaire to your channel; open the channel section and configure your questionnaire on the default channel if you have only that.

**Reply by Gianca77** [reply]
Dear @evilaliv3 
Exactly!! now it works, I'm sorry if I open this stupid issue without a deep check but I searched a bit and only found the solution after opening the ticket
Thank You!

**Comment by ximo99** [comment]
Hello,

I have a new question. Is it possible to have two channels available? I explain. Currently, when the user clicks 'Submit a report', I go to the quiz I have in the 'Default' channel. This quiz is currently called 'Questionary 1'.

Would it be possible to have 2 channels available with different questionnaires, called 'Questionary 1' and 'Questionary 2', and for the user to choose which one to send? How can I change the channel that is displayed when I click 'Submit a report' from the one in the 'Default' channel to another channel?

Thank you very much for your work, it is excellent.
Greetings.
